### PR TITLE
Update readme files + Update the Python Cost Reduction Tool

### PR DESCRIPTION
**Pull request by BotrosHanna-INL**
--------
Merge Request Description
--------
**What issue does this change request address?**

**What are the changes due to this change request?**

----------------
For Change Request Review
----------------
- [x] 1. Is the merge request reference an issue?  If the issue is closed, the issue close checklist shall be done.
- [x] 2. Is the merge request reference a task?  If the task is fulfilled, the task close checklist shall be done.
- [x] 3. If any changes to the database, there must be an accompanying change to the all the linked tables.

### CODE REVIEW

**Review by JiaZhou-PU — COMMENTED** [review]
approve

File: README.md [hunk 1]
```diff
@@ -137,3 +137,10 @@ ACCERT can be executed both through the Workbench interface by pressing `run` or
 `python ACCERT/src/Main.py -i myinput.son`
 
 
+
+# A New Tool is implemented in ACCERT to Quantify Capital Cost Reduction Pathways for Advanced Nuclear Reactors
```
Comment (cbolisetti) [inline README.md:141]: ``` # A Tool for Quantifying Capital Cost Reduction Reduction Pathways for Advanced Nuclear Reactors ```

File: README.md [hunk 2]
```diff
@@ -137,3 +137,10 @@ ACCERT can be executed both through the Workbench interface by pressing `run` or
...
+- A tool to showcase different pathways to reducing the capital cost of advanced reactor plants.
```
Comment (cbolisetti) [inline README.md:142]: A new tool has been developed to simulate the different pathways to reduce the costs of advanced nuclear power plants when constructed through large order books. This tool inputs various levers that represent high-level decision making and outputs the capital costs and construction duration starting from the first plant to the last plant in the order book.

File: README.md [hunk 3]
```diff
@@ -137,3 +137,10 @@ ACCERT can be executed both through the Workbench interface by pressing `run` or
...
+- The methodology and assumptions behind this tool are explained in a recently published report: https://inldigitallibrary.inl.gov/sites/sti/sti/Sort_109810.pdf
```
Comment (cbolisetti) [inline README.md:143]: ```- The methodology and assumptions behind this tool are documented [here](https://inldigitallibrary.inl.gov/sites/sti/sti/Sort_109810.pdf]).   ```

File: README.md [hunk 4]
```diff
@@ -137,3 +137,10 @@ ACCERT can be executed both through the Workbench interface by pressing `run` or
...
+- There are two versions of this tool
+	1. The Python version (notebooks) : is found in the folder: "Cost_Reduction"
+	2. The excel version :  is found in the folder: "excel_framework"
```
Comment (cbolisetti) [inline README.md:146]: An excel spreadsheet that can be found [here](https://fuelcycleoptions.inl.gov/Shared%20Documents/Nuclear-Reactor-Capital-Cost-Reduction-Pathway-Tool.pdf).

File: README.md [hunk 5]
```diff
@@ -137,3 +137,10 @@ ACCERT can be executed both through the Workbench interface by pressing `run` or
 `python ACCERT/src/Main.py -i myinput.son`
 
 
+
+# A New Tool is implemented in ACCERT to Quantify Capital Cost Reduction Pathways for Advanced Nuclear Reactors
+- A tool to showcase different pathways to reducing the capital cost of advanced reactor plants.
+- The methodology and assumptions behind this tool are explained in a recently published report: https://inldigitallibrary.inl.gov/sites/sti/sti/Sort_109810.pdf
+- There are two versions of this tool
+	1. The Python version (notebooks) : is found in the folder: "Cost_Reduction"
```
Comment (cbolisetti) [inline README.md:145]: A Python Jupyter Notebook in the folder, "Cost_Reduction".

File: README.md [hunk 6]
```diff
@@ -137,3 +137,10 @@ ACCERT can be executed both through the Workbench interface by pressing `run` or
 `python ACCERT/src/Main.py -i myinput.son`
 
 
+
+# A New Tool is implemented in ACCERT to Quantify Capital Cost Reduction Pathways for Advanced Nuclear Reactors
+- A tool to showcase different pathways to reducing the capital cost of advanced reactor plants.
+- The methodology and assumptions behind this tool are explained in a recently published report: https://inldigitallibrary.inl.gov/sites/sti/sti/Sort_109810.pdf
+- There are two versions of this tool
```
Comment (cbolisetti) [inline README.md:144]: colon after 'tool'

**Review by cbolisetti — CHANGES_REQUESTED** [review]

**Comment by cbolisetti** [comment]
@BotrosHanna-INL and @JiaZhou-PU  can you please also move the costreduction stuff to the end of the getting started section? Also at the end of the description, please add the following: 
"As currently implemented, this tool is separate from the ACCERT cost estimation tool. In the future, the ACCERT cost estimation and cost reduction tools may be integrated so that users can estimate the costs of their own design using the estimation tool and use the cost reduction tool to simulate the evolution of its capital costs from FOAK to NOAK plants. "

File: Cost_Reduction/README.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+## A Tool to Quantify Capital Cost Reduction Pathways for Advanced Nuclear Reactors
+- This is a Python tool that estimated the impact of several variables on the capital cost of the nuclear reactors.
+- The methodology and assumptions behind this tool are explained in detail in this report:  https://inldigitallibrary.inl.gov/sites/sti/sti/Sort_109810.pdf
+- Another excel tool for the cost reduction can be found here: https://fuelcycleoptions.inl.gov/Shared%20Documents/Nuclear-Reactor-Capital-Cost-Reduction-Pathway-Tool.pdf
```
[the remark this answers is not in the document]
Comment (BotrosHanna-INL) [reply Cost_Reduction/README.md:4]: Done

File: Cost_Reduction/README.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+## A Tool to Quantify Capital Cost Reduction Pathways for Advanced Nuclear Reactors
+- This is a Python tool that estimated the impact of several variables on the capital cost of the nuclear reactors.
```
[the remark this answers is not in the document]
Comment (BotrosHanna-INL) [reply Cost_Reduction/README.md:2]: Dibe

File: Cost_Reduction/README.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
+- The methodology and assumptions behind this tool are explained in detail in this report:  https://inldigitallibrary.inl.gov/sites/sti/sti/Sort_109810.pdf
+- Another excel tool for the cost reduction can be found here: https://fuelcycleoptions.inl.gov/Shared%20Documents/Nuclear-Reactor-Capital-Cost-Reduction-Pathway-Tool.pdf
+- This tool includes the following files
+	1. "CostReduction_exploration_mode.ipynb": a file that demonstrated how the capital cost (structured as a code of account) will change step by step due to each single variable
```
[the remark this answers is not in the document]
Comment (BotrosHanna-INL) [reply Cost_Reduction/README.md:6]: Done

File: README.md (hunk 1 above)
Comment (BotrosHanna-INL) [reply README.md:141]: Done.

File: README.md (hunk 2 above)
Comment (BotrosHanna-INL) [reply README.md:142]: Done

File: README.md (hunk 3 above)
Comment (BotrosHanna-INL) [reply README.md:143]: Done.

File: README.md (hunk 4 above)
Comment (BotrosHanna-INL) [reply README.md:146]: Done.

File: README.md (hunk 5 above)
Comment (BotrosHanna-INL) [reply README.md:145]: Done.

File: README.md (hunk 6 above)
Comment (BotrosHanna-INL) [reply README.md:144]: Done

**Comment by BotrosHanna-INL** [comment]
I made all the changes.
For the last comment, the cost reductio stuff are already at the end of getting started section.

**Review by cbolisetti — APPROVED** [review]

File: README.md [hunk 10]
```diff
@@ -137,3 +137,13 @@ ACCERT can be executed both through the Workbench interface by pressing `run` or
 `python ACCERT/src/Main.py -i myinput.son`
 
 
+
+# A Tool for Quantifying Capital Cost Reduction Reduction Pathways for Advanced Nuclear Reactors
+- A new tool has been developed to simulate the different pathways to reduce the costs of advanced nuclear power plants when constructed through large order books. This tool inputs various levers that represent high-level decision making and outputs the capital costs and construction duration starting from the first plant to the last plant in the order book.
+
+-The methodology and assumptions behind this tool are documented [here]: https://inldigitallibrary.inl.gov/sites/sti/sti/Sort_109810.pdf
```
Comment (cbolisetti) [inline README.md:144]: Looks like there's a typo here..the hyperlink may not show up

Comment (BotrosHanna-INL) [reply README.md]: I just fixed the typos. Also, the hyperlink will work. I can see that it is working on on ACCERT Repo:
https://github.com/BotrosHanna-INL/ACCERT

**Comment by cbolisetti** [comment]
> I made all the changes.
> 
> For the last comment, the cost reductio stuff are already at the end of getting started section.

Thanks, @BotrosHanna-INL

**Comment by cbolisetti** [comment]
@JiaZhou-PU looks like you still need to approve this PR before we can merge it. Can you please approve?

**Review by JiaZhou-PU — APPROVED** [review]
approve

**Pull request merged**